Assigment 1/Example1.java

```diff
@@ -0,0 +1,16 @@
+import java.io.Console;
+import java.lang.Math;
+
+public class Example1 {
+    
+    public static void main(String [] args) {
+        Console console = System.console();
+        System.out.println("Largest Number");
+        double digit = Double.parseDouble(console.readLine());
+        System.out.println(largestNumber(digit));
+    }
+
+    public static double largestNumber(final double digits) {
```

Comment:
Awesome, but this will need to return an integer as we only need the number avoiding to have the decimal section